### DISCUSSION TITLE
How do I format text dynamically?

### DISCUSSION THREAD

**Discussion by ghost** [singerdmx/flutter-quill#737] [General]
I want to format text within the editor by changing it's background colour, but dynamically. How can I do this?  

I came across this one function, ```formatText(int index, int len, Attribute<dynamic>? attribute)``` within the controller, but I am confused as to what the ```attribute``` is. I also tried to make the text bold by passing ```Attribute.bold```, and it worked. But now I wish to change the background colour. I have no idea how to do this.

Any help is appreciated.

**Comment by wei-gtstudio** [comment]
Is this fit your requirements ?

```
    QuillEditor(
      customStyleBuilder: ,
      customStyles: ,
    );
```

**Reply by ghost** [reply]
Yep, it works. Thank you.
